### DISCUSSION TITLE
The animation utility does not seem to be working

### DISCUSSION THREAD

**Discussion by ajmeraaxesh** [tailwindlabs/tailwindcss#2073] [Help]
This is my tailwind.config.js. I am using the animation utility as shown in the PR. Have I set it correctly or I am missing out on something. I am using tailwind.css v1.5.2

module.exports = {
    theme: {
        extend: {
            colors: {
                'brand-blue': '#369dd6',
              
            },
            fontFamily: {
                sans: ['Inter var', ...defaultTheme.fontFamily.sans],
           
            },
        },
        animation: {
            none: 'none',
            spin: 'spin 1s linear infinite',
            ping: 'ping 1s cubic-bezier(0, 0, 0.2, 1) infinite',
            pulse: 'pulse 2s cubic-bezier(0.4, 0, 0.6, 1) infinite',
            bounce: 'bounce 1s infinite',
        },
        keyframes: {
            spin: {
                from: { transform: 'rotate(0deg)' },
                to: { transform: 'rotate(360deg)' },
            },
            ping: {
                '0%': { transform: 'scale(1)', opacity: '1' },
                '75%, 100%': { transform: 'scale(2)', opacity: '0' },
            },
            pulse: {
                '0%, 100%': { opacity: '1' },
                '50%': { opacity: '.5' },
            },
            bounce: {
                '0%, 100%': {
                    transform: 'translateY(-25%)',
                    animationTimingFunction: 'cubic-bezier(0.8,0,1,1)',
                },
                '50%': {
                    transform: 'translateY(0)',
                    animationTimingFunction: 'cubic-bezier(0,0,0.2,1)',
                },
            },
        },
    },
    variants: {
        animation: ['responsive']
    },
    plugins: [
       
    ],
    purge: [
        './src/**/*.html',
        './src/**/*.vue',
        './src/**/*.jsx',
        './src/**/*.js',
        './public/**/*.js',
        './public/**/*.html',
    ],
}

**Comment by estevanmaito** [comment]
Notice that the PR https://github.com/tailwindlabs/tailwindcss/pull/2068 is still open. It's not part of Tailwind yet.

**Reply by ajmeraaxesh** [reply]
I am under the impression, that this was the reason, It got tweeted

**Reply by ajmeraaxesh** [reply]
https://twitter.com/adamwathan/status/1286695715482341377
